skyreader/result.py

```diff
@@ -1235,38 +1189,25 @@ def circular_contour(ra, dec, sigma, nside):
         print("Contour Area (90%):", contour_areas[1], "degrees (cartesian)",
             healpy_area, "degrees (scaled)")
 
-        if dosave:
-            # Dump the whole contour
-            path = unique_id + ".contour.pkl"
-            print("Saving contour to", path)
-            with open(path, "wb") as f:
-                pickle.dump(saving_contours, f)
 
-            healpy.write_map(f"{unique_id}.skymap_nside_{mmap_nside}.fits.gz",
-                equatorial_map, coord = 'C', column_names = ['2LLH'],
-                extra_header = fits_header, overwrite=True)
+        # Dump the whole contour
+        path = unique_id + ".contour.pkl"
+        print("Saving contour to", path)
```

Comment:
You have several `print`s. Generally, I recommend avoiding this pattern for anything other than scripts (I see this package as a library) and using logs instead. What's the intended use of these plotting functions?

Reply:
Yes, I don't like it either, it is a pattern inherited from the old code. Surely we can implement proper loggers.

Reply:
I would say this is a todo for a future PR.